extensions/_states/vault.py

```diff
@@ -134,6 +134,45 @@ def audit_backend_enabled(name, backend_type, description='', options=None,
     return ret
 
 
+def secret_backend_enabled(name, backend_type, description='',
+                           mount_point=None, config=None):
+    backends = __salt__['vault.list_secret_backends']().get('data', {})
```

Comment:
Why did u use .get('data' ... in the secret backend but not in the auth_backend_enabled or the audit_backend_enabled functions?